keel-slack/src/main/kotlin/com/netflix/spinnaker/keel/slack/SlackService.kt

```diff
@@ -45,4 +46,17 @@ class SlackNotifier(
       log.debug("new slack integration is not enabled")
     }
   }
+
+  fun getUserInfoByEmail(email: String): String? {
```

Comment:
This is not really returning user info but a user "handle". Consider renaming:
```suggestion
  fun getUserHandleByEmail(email: String): String? {
```

Reply:
what does `user handle` means? 🤔

Reply:
done